packages/flutter/lib/src/rendering/proxy_sliver.dart

```diff
@@ -7,6 +7,7 @@ import 'dart:ui' as ui show Color;
 import 'package:flutter/foundation.dart';
 import 'package:vector_math/vector_math_64.dart';
 
+import '../rendering/layer.dart';
```

Comment:
SHouldn't this just be 

```suggestion
import 'layer.dart';
```

Reply:
yes, we should never import with `../foo/bar.dart`

Reply:
Oops. Done